tests/test_cli/test_push.py

```diff
@@ -207,3 +211,31 @@ def test_push_contract_registry_positive(self, *mocks):
             standalone_mode=False,
         )
         self.assertEqual(result.exit_code, 0)
+
+
+class TestPushLocallyWithLatest(AEATestCaseEmpty):
+    """Test push locally with 'latest' as version."""
+
+    @pytest.mark.parametrize("component_type", ITEM_TYPES)
+    def test_command(self, component_type):
+        """Run the test."""
+        item_name = f"my_{component_type}"
+        self.scaffold_item(component_type, item_name)
```

Comment:
what could be used here instead of scaffolding an item? Maybe using `dummy_aea` as agent?

Reply:
This is fine.